plugin.py

```diff
@@ -30,19 +36,19 @@ def start_julia_repl(window: sublime.Window, focus: bool) -> None:
     window.run_command("terminus_open", {
         "cmd": cmd,
         "cwd": "${file_path:${folder}}",
-        "panel_name": JULIA_REPL_NAME,
+        "title": panel and "" or JULIA_REPL_NAME,
+        "panel_name": panel and JULIA_REPL_NAME or "",
```

Comment:
The "title" argument that is passed to Terminus is what is returned in `view.name()`.
`"title"` undefined if `panel` is `True`
`"panel_name"` undefined if `panel` is `False`